### PR TITLE
[RFR][NOTEST] Added cleanup for SQS queues

### PR DESCRIPTION
**Pull request by mmojzis**
Deleting SQS queues older than 14 days.
This wrapanapi PR required for this PR: https://github.com/ManageIQ/wrapanapi/commit/291b029fdcc5860218201d694f3408a3a76289b9
Also after merge please keep rerunning ec2cleanup for at least one day as there are so many queues and only 1k are deleted by one run. @mshriver

### CODE REVIEW

File: scripts/ec2cleanup.py [hunk 1]
```diff
@@ -189,8 +192,34 @@ def delete_snapshots(provider_mgmt, excluded_snapshots, output):
...
+    current_timestamp = time.time()
+    try:
+        queues = provider_mgmt.list_queues_with_creation_timestamps()
+        for queue in queues:
+            if excluded_queues and queue in excluded_queues:
+                logger.info(" Excluding Queue with url: %r", queue)
+                continue
+            else:
+                # 1209600 is 14 days in seconds
+                if float(current_timestamp) - float(queues[queue]) > 1209600:
```
Comment (digitronik) [inline scripts/ec2cleanup.py]: Required:
`current_timestamp` will always `float` value. please remove conversion.

File: scripts/ec2cleanup.py [hunk 2]
```diff
@@ -189,8 +192,34 @@ def delete_snapshots(provider_mgmt, excluded_snapshots, output):
...
+                    provider_mgmt.delete_sqs_queue(queue)
+                    queue_list.append([provider_name, queue])
+        logger.info("  Deleted Queues: %r", queue_list)
+        with open(output, 'a+') as report:
+            if queue_list:
```
Comment (digitronik) [inline scripts/ec2cleanup.py]: Optional:
I would open a file for writing only if `queue_list` available.

File: scripts/ec2cleanup.py [hunk 3]
```diff
@@ -215,6 +244,10 @@ def ec2cleanup(exclude_volumes, exclude_eips, exclude_elbs, exclude_enis, exclud
                       excluded_stacks=exclude_stacks,
                       stack_template=stack_template,
                       output=output)
+        logger.info("Deleting old queues...")
```
Comment (digitronik) [inline scripts/ec2cleanup.py]: :+1:

File: scripts/ec2cleanup.py (hunk 2 above)
Comment (mmojzis) [reply scripts/ec2cleanup.py]: You are right. That makes sense.

File: scripts/ec2cleanup.py (hunk 1 above)
Comment (mmojzis) [reply scripts/ec2cleanup.py]: Fixed.

**Pull request merged**